modules/ROOT/pages/_partials/cookbook-dw/map-an-object-key/out.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is this output alright? You're loosing the values coming from the payload

Reply:
Hi @teofr Could you help me updating the DW script to return the full payload with the appended new object to the matching sheet?

Reply:
I think the script is alright, but is not working properly, can you hold merging this PR for a bit, I'm checking what can be the issue

Reply:
Already updated. Thanks again.